coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -80,7 +80,6 @@ RDEPEND="${RDEPEND}
 		app-emulation/xenstore
 		net-fs/cifs-utils
 		sys-auth/realmd
-		sys-auth/sssd
```

Comment:
By removing this item from the list, it won't be installed into any image. Instead you would need to move the entry into the list starting at 86 (and let's keep it sorted)

Reply:
Updated. Thanks for your patience as I learn. :)